src/modules/Blend/info.json

```diff
@@ -1,17 +1,34 @@
 {
   "name": "blend",
-  "description": "Blend two chosen image steps with the given function. Defaults to using the red channel from image 1 and the green and blue and alpha channels of image 2. Easier to use interfaces coming soon!",
+  "description": "Blend two chosen image steps with the given function. Defaults to using the red channel from image 1 and the green and blue and alpha channels of image 2.",
   "inputs": {
     "offset": {
       "type": "integer",
       "desc": "Choose which image to blend the current image with. Two steps back is -2, three steps back is -3 etc.",
       "default": -2
     },
+    "blendMode": {
+      "type": "select",
+      "desc": "Name of the Blend Mode to use",
+      "default": "default",
```

Comment:
Doesn't the default blend mode have a name?

Reply:
No, by default it uses the user defined function for blending two images, Should I change its name to something else?

Reply:
What will happen if the user selects this option?

Reply:
When this option is selected then the function input field(which was initially there when blend modes were not added) is used to determine the output image.

Reply:
Oh. So can it be called `custom` or `User Input` or something?

Reply:
Yes that makes more sense :sweat_smile: